src/EventDelegate/index.ts

```diff
@@ -49,7 +49,7 @@ export class EventDelegate {
     if (popEngine.isPopForTrigger(target, trigger)) {
       if (popEngine.isPopAlreadyOpenForTarget(target)) {
         let groupId = target.getAttribute('popgun-group');
-        timeoutManager.maybeClearTimeout(timeoutManager.getTimeouts().timeToHoverOnPop, groupId);
+        timeoutManager.maybeClearTimeout(popEngine._timeouts.timeToHoverOnPop, groupId);
```

Comment:
I do this over here:
https://github.com/relateiq/popgun/pull/34/files#diff-73085523373ecb1eea19262129e4badaR31